src/constants/index.js

```diff
@@ -1,5 +1,6 @@
 module.exports = Object.freeze({
   BLANK_FIELD: '--',
+  CHART_TOOLTIP_DATE_FORMAT: 'mmm d, yyyy',
```

Comment:
Probably should do DATE_FORMAT_CHART_TOOLTIP. Same for the X axis one below (sorry, I know that's not in this PR)

Reply:
Then again it's not the worst it starts with "chart". Your call